README.md

```diff
@@ -4,6 +4,6 @@
 
 This R Interface to C API of IBM ILOG CPLEX provides CPLEX support for [sybil](https://github.com/SysBioChalmers/sybil), and requires IBM ILOG CPLEX version >= 12.1 .
 
-If you're facing any problems, please open an [issue](https://github.com/SysBioChalmers/sybil-cplexAPI/issues/new) or a [discussion](https://github.com/SysBioChalmers/sybil-cplexAPI/discussions/new).
```

Comment:
Just a note - with some renaming magic, both the old links and the new ones work.